packages/relay-kit/src/packs/safe-4337/estimators/PimlicoFeeEstimator.ts

```diff
@@ -22,6 +22,7 @@ export class PimlicoFeeEstimator implements IFeeEstimator {
 
   async adjustEstimation({ userOperation }: EstimateFeeFunctionProps): Promise<EstimateGasData> {
     return {
+      callGasLimit: userOperation.callGasLimit + userOperation.callGasLimit / 2n,
```

Comment:
As a temporary workaround we increase the `callGasLimit` value by 50% to avoid  `out of gas` errors.